.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+Dockerfile @eriner
+
+### Security Sensitive ###
+.github/workflows/ @lukechampine
```

Comment:
I don't think a CODEOWNERS file is necessary for a project of this size. I will review every pull request.